app/views/pages/contact_us.html.erb

```diff
@@ -32,53 +32,67 @@
       <h2 class="govuk-heading-m">Contact us</h2>
       <p class="govuk-body">Complete this form for technical help</p>
 
-      <%= form_tag(contact_us_path, method: :post, id: "help_form") do %>
+      <%= form_for @contact, url: contact_us_path, method: :post, id: "help_form" do |f| %>
         <div class="<%= field_error(@contact.errors, :name) %>">
-          <%= label_tag "Full name", "Full name", class: 'govuk-label' %>
+          <%= f.label "Full name", "Full name", class: 'govuk-label' %>
           <% if @contact.errors[:name].present? %>
                   <span class="govuk-error-message">
                     <%= @contact.errors[:name].first %>
                   </span>
+            <%= f.text_field("name", id: "name", class: "govuk-input govuk-input--error") %>
+          <% else %>
+            <%= f.text_field("name", id: "name", class: "govuk-input") %>
           <% end %>
-          <%= text_field_tag("name", @contact.name, id: "name", class: "govuk-input") %>
         </div>
+
         <div class="<%= field_error(@contact.errors, :email_address) %>">
-          <%= label_tag "email_address", "Email address", class: 'govuk-label' %>
+          <%= f.label "email_address", "Email address", class: 'govuk-label' %>
           <% if @contact.errors[:email_address].present? %>
                 <span class="govuk-error-message">
                   <%= @contact.errors[:email_address].first %>
                 </span>
+            <%= f.email_field("email_address", id: "email_address", class: "govuk-input govuk-input--error") %>
+          <% else %>
+            <%= f.email_field("email_address", id: "email_address", class: "govuk-input") %>
```

Comment:
Is there a better way of including `govuk-input--error` when there's an error?  I think the duplication is going to susceptible for errors if it changes and someone forgets to do both.